ASP.NET Core/Web/AspNetCoreTemplate.Web/AspNetCoreTemplate.Web.csproj

```diff
@@ -18,13 +18,13 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.1.1" />
```

Comment:
Do we need this package

Reply:
This is used for scaffolding. Might be removed.